system-tests/__snapshots__/browser_crash_handling_spec.js

```diff
@@ -217,28 +220,33 @@ exports['Browser Crash Handling / when the browser process crashes in chrome / f
   ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
   │ Cypress:    1.2.3                                                                              │
   │ Browser:    FooBrowser 88                                                                      │
-  │ Specs:      2 found (chrome_process_crash.cy.js, simple.cy.js)                                 │
-  │ Searched:   cypress/e2e/chrome_process_crash.cy.js, cypress/e2e/simple.cy.js                   │
+  │ Specs:      2 found (chrome_tab_crash.cy.js, simple.cy.js)                                     │
+  │ Searched:   cypress/e2e/chrome_tab_crash.cy.js, cypress/e2e/simple.cy.js                       │
   └────────────────────────────────────────────────────────────────────────────────────────────────┘
 
 
 ────────────────────────────────────────────────────────────────────────────────────────────────────
                                                                                                     
-  Running:  chrome_process_crash.cy.js                                                      (1 of 2)
+  Running:  chrome_tab_crash.cy.js                                                          (1 of 2)
 
 
 
-We detected that the Chrome process just crashed with code 'null' and signal 'SIGTRAP'.
+We detected that the Chromium Renderer process just crashed.
```

Comment:
I wonder why this changed :thinking:

Reply:
Oh I see, I think the snapshots in the file are just in a different order now.

Reply:
Getting these snapshots to update correctly was a bit of a challenge 😅 . Sometimes the snapshots do update in a different order but should be the same relative to their export.

I noticed we have a conflict in `browser_crash_handling_spec` from the [improved memory work](https://github.com/cypress-io/cypress/commit/d1dc1258eafa6ff047abc913ebb930984e02a38d#diff-4c495b06189c0abcf578534fcb0e4f030df90cc9c4b3930c5923b61dfbb7025cR29). I am rebuilding that snapshot now to fix the conflict and will update @zachz-indeed 's branch shortly